src/Views/Components/ListTable/ListTableRows/index.tsx

```diff
@@ -0,0 +1,108 @@
+import styles from './ListTableRows.module.scss';
+import {__} from '@wordpress/i18n';
+import cx from 'classnames';
+import {useEffect, useState} from 'react';
+import TableCell from '../TableCell';
+import {BulkActionCheckbox} from '@givewp/components/ListTable/BulkActions/BulkActionCheckbox';
+import InterweaveSSR from '@givewp/components/ListTable/InterweaveSSR';
+
+//@unreleased determines if row should display actions based on column.id
+const displayRowActions = (column) => {
+    switch (column.id) {
+        case 'id':
+            return true;
+        case 'donorInformation':
+            return true;
+        default:
+            return false;
+    }
+};
```

Comment:
Can we have the RowActions component on every first column instead? We'd be preventing issues in the future when we add the ability to change order and/or hide columns.

Reply:
Resolved: https://github.com/impress-org/givewp/commit/dba28cd69f894b6fa199240c6de7658851e96e90